code/ConvertToStandardPath_SubmarineCable.py

```diff
@@ -295,38 +285,95 @@ def calculate_distance_between_cable_landing_points(wktToCities, cableIdToWkt):
             else:
-                if not start_city in graph:
-                    fail_cities.add(start_city)
-                    print(f"city {city_pair[0][0]}, {start_city} not exist in graph for {cable_id}.")
+                if not start_city_coord in graph:
+                    fail_cities.add(start_city_coord)
+                    print(f"city {start_city_name}, {start_city_coord} not exist in graph for {cable_id}.")
                 else:
-                    fail_cities.add(end_city)
-                    print(f"city {city_pair[1][0]}, {end_city} not exist in graph for {cable_id}.")
+                    fail_cities.add(end_city_coord)
+                    print(f"city {end_city_name}, {end_city_coord} not exist in graph for {cable_id}.")
+    return submarine_standard_paths
 
+def insert_submarine_standard_paths_to_database(db_file, submarine_standard_paths):
+    conn = sqlite3.connect(db_file)
+    cursor = conn.cursor()
+
+    # Check if the table exists
+    cursor.execute("SELECT name FROM sqlite_master WHERE type='table' AND name='submarine_standard_paths';")
+    table_exists = cursor.fetchone()
+
+    # If the table exists, drop it
+    if table_exists:
+        cursor.execute("DROP TABLE submarine_standard_paths;")
+
+    # print("Table dropped")
+
+    cursor.execute("CREATE TABLE submarine_standard_paths ( \
+        from_city TEXT, \
+        from_state TEXT, \
+        from_country TEXT, \
+        to_city TEXT, \
+        to_state TEXT, \
+        to_country TEXT, \
+        distance_km REAL, \
+        path_wkt TEXT \
+    );")
+
+    # print("Table created.")
+
+    for submarine_standard_path in submarine_standard_paths:
+        # Execute insert query
+                cursor.execute("INSERT INTO submarine_standard_paths (from_city, from_state, from_country, to_city, to_state, to_country, distance_km, path_wkt) VALUES (?, ?, ?, ?, ?, ?, ?, ?)", submarine_standard_path)
+                # print("One data inserted")
+                
+    conn.commit()
+    conn.close()
+
+def submarine_standard_path_table_tester(db_file):
+    conn = sqlite3.connect(db_file)
+    cursor = conn.cursor()
+    sql_query = """
+    SELECT ssp.from_city, ssp.from_state, ssp.from_country, ssp.to_city, ssp.to_state, ssp.to_country, ssp.distance_km, ssp.path_wkt
+    FROM submarine_standard_paths ssp
+    """
+    cursor.execute(sql_query)
+    datas = cursor.fetchall()
+    for data in datas:
+        print(data)
+    conn.close()
 
 if __name__ == "__main__":
-    wktToCities, cableIdToWkt =  get_data_from_database('../database/igdb.db')
-    calculate_distance_between_cable_landing_points(wktToCities, cableIdToWkt)
-    
+    db_file = '../database/igdb.db'
+    wktToCities, cableIdToWkt =  get_data_from_database(db_file)
+    submarine_standard_paths = calculate_distance_between_cable_landing_points(wktToCities, cableIdToWkt)
+    print(len(submarine_standard_paths))
```

Comment:
If this is just for debugging, remove it. Otherwise, add properly description rather than just printing a number. No user (including you after two months) can understand what this number is by just running this script.

Reply:
That's true, it is used for debugging. I also verify other print in this code to ensure there is no other mysterious print in for this code.